files/generate_vars.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+import configparser
```

Comment:
Does this work in both python2 and 3? 2 has ConfigParser. You can do e.g.:

try:
    import configparser
except ImportError:
    import ConfigParser as configparser

Reply:
I will fix that but
Besides that, we have many issues regarding to python compatibility version (For example raw_input vs input)
I've discussed this with Ondra and currently we will support only python 2 and when we port all ansble cpde to python 3  we will support this python script as well.